kview/src/main/resources/dev/ikm/komet/kview/mvvm/view/kview.css

```diff
@@ -2382,30 +2382,103 @@ Timeline Year line segment (will overlay change circles)
     -fx-font-family: "Noto Sans";
 }
 
-.blue-arrow-combo-box-grey11 .list-cell {
+/* date selection combo box used by export */
+.blue-arrow-combo-box-grey11.combo-box-base {
     -fx-text-fill: -Grey-11;
+    -fx-background-color: -Grey-1;
+    -fx-border-color: -Grey-5;
+    -fx-border-radius: 4;
+
     -fx-font-family: "Noto Sans";
     -fx-font-size: 14px;
     -fx-font-style: normal;
-    -fx-font-weight: 500;   
-    -fx-background-color: -Grey-1;
-}
+    -fx-font-weight: 500;
 
-.blue-arrow-combo-box-grey11.combo-box-base {
-    -fx-border-radius: 4;
-    -fx-border-color: -Grey-5;
-    -fx-background-color: -Grey-1;
+    /*-width-size: 180px;*/
```

Comment:
Can we remove -width-size: 180px; 
since its commented?

Reply:
yes